content/en/docs/instrumentation/java/automatic/servers.md

```diff
@@ -0,0 +1,137 @@
+---
```

Comment:
Maybe the file should be renamed `server-config.md`?

Reply:
+1